static/app/views/dashboardsV2/create.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router';
+
+import {Organization} from 'app/types';
+import withOrganization from 'app/utils/withOrganization';
+
+import {EMPTY_DASHBOARD} from './data';
+import DashboardDetail from './detail';
+import {DashboardState} from './types';
+import {cloneDashboard} from './utils';
+
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  organization: Organization;
+  children: React.ReactNode;
+};
+
+class CreateDashboard extends React.Component<Props> {
```

Comment:
should we convert this class component to functional component since you are not using state here?